packages/@lwc/compiler/src/options.ts

```diff
@@ -14,6 +14,7 @@ type RecursiveRequired<T> = {
 const DEFAULT_OPTIONS = {
     isExplicitImport: false,
     preserveHtmlComments: false,
+    disableStaticContentOptimization: false,
```

Comment:
I just realized this flag is at the compiler level and not at the template compiler. Would it make sense to course correct this now at the LWC template compiler level as well since we recently merged this PR?

Reply:
That makes sense to me, I named it enableStaticContentOptimization to match the gate definition name.